src/views/OtherTools.md

```diff
@@ -6,21 +6,36 @@
         <p>
                 <a href="http://monarch-exomiser-web-dev.monarchinitiative.org/exomiser">Exomiser</a>
                 is a Java program that functionally annotates variants
-                from whole-exome sequencing data starting from a VCF file (version 4).
-                The functional annotation code is based on Jannovar and uses UCSC
+                from whole-exome sequencing data starting from a VCF file (version 4) and a set 
+                of phenotypes encoded using the <a href="https://hpo.jax.org/app/">Human Phenotype 
+                Ontology</a>. The functional annotation of variants is based on Jannovar and uses UCSC
                 KnownGene transcript definitions and hg19 genomic coordinates.<br>
                 Variants are prioritized according to user-defined criteria on variant
                 frequency, pathogenicity, quality, inheritance pattern, and model
-                organism phenotype data. Predicted pathogenicity data was extracted
+                organism phenotype data. Predicted pathogenicity data is extracted
                 from the dbNSFP resource. Cross-species phenotype comparisons are
                 powered by the <a href="http://owlsim.org">OWLSim algorithm</a>.<br>
-                Exomiser is currently using mouse phenotypes and will soon be leveraging
+                Exomiser is currently using mouse phenotypes and will soon leverage
```

Comment:
We do human, mouse and fish phenotypes. The rest is completely untrue. It might happen, it might not.

Reply:
Thanks, updated